### DISCUSSION TITLE
Any other exploits??

### DISCUSSION THREAD

**Discussion by Arthura0121** [3kh0/ext-remover#577] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
I'm currently on version 109 and I was wondering if there were any working exploits other than Sh1mmer or downgrading...
Thanks! :)

**Comment by S-PScripts** [comment]
https://github.com/S-PScripts/chromebook-utilities/blob/main/Exploits/Side%20Panel%20Webview%20(v108-v112)

**Comment by schoolexploitkid** [comment]
LoMoH (Locked Mode Hack)

**Comment by Charlieis1nsan3** [comment]
https://github.com/3kh0/ext-remover/discussions/1258
